django/views/csrf.py

```diff
@@ -1,10 +1,9 @@
 from django.conf import settings
 from django.http import HttpResponseForbidden
-from django.template import Context, Engine
+from django.template import Context, Engine, TemplateDoesNotExist, loader
 from django.utils.translation import ugettext as _
 from django.utils.version import get_docs_version
 
-
```

Comment:
Ok, I'll do so next time. I didn't want to leave this PR hanging around with a "CI failing" flag ;) (At least I remember that either isort or flake8 were complaining.)